### DISCUSSION TITLE
[BUG] phpmyadmin 登录后跳转到新的url 报错404

### DISCUSSION THREAD

**Discussion by rockerdance** [1Panel-dev/1Panel#7043] [Q&A]
### 1Panel 版本

v1.10.20-lts

### 问题描述

在phpmyadmin 登录后，自动跳转到：

http://domain.com:8980/api/direct?url=aHR0cDovL2RvbWFpbi5jb206ODY4OS9pbmRleC5waHA/cm91dGU9LyZyb3V0ZT0lMkY=

然后页面报错404
Not Found
The requested URL was not found on this server.

Apache/2.4.62 (Debian) Server at domain.com Port 8980

### 重现步骤

1.进入phpmyadmin 管理：
http://domain.com:8980/
2.输入：
服务器：domain.com
用户名： xxxx
密码： password
3.点击登录
4.页面跳转：http://domain.com:8980/api/direct?url=aHR0cDovL2RvbWFpbi5jb206ODY4OS9pbmRleC5waHA/cm91dGU9LyZyb3V0ZT0lMkY=

### 相关日志输出

```shell
- - [14/Nov/2024:01:37:03 +0000] "POST /index.php?route=/ HTTP/1.1" 302 1061 "-" "Mozilla/5.0 (Macintosh; Intel Mac OS X 10.15; rv:132.0) Gecko/20100101 Firefox/132.0"
- - [14/Nov/2024:01:37:03 +0000] "GET /api/direct?url=aHR0cDovL2RvbWFpbi5jb206ODY4OS9pbmRleC5waHA/cm91dGU9LyZyb3V0ZT0lMkY= HTTP/1.1" 404 498 "-" "Mozilla/5.0 (Macintosh; Intel Mac OS X 10.15; rv:132.0) Gecko/20100101 Firefox/132.0"
  | 127.0.0.1 - - [14/Nov/2024:01:37:04 +0000] "OPTIONS * HTTP/1.0" 200 126 "-" "Apache/2.4.62 (Debian) PHP/8.2.25 (internal dummy connection)"
```

**Comment by wanghe-fit2cloud** [comment]
在本地通过 IP:Port 和反向代理方式访问时，登录页面输入 MySQL 数据库信息后，均未复现该问题。建议将此问题反馈至 phpMyAdmin 官方，看看是否有相关反馈或解决方案。
